src/overlays/actors/ovl_Obj_Hsblock/z_obj_hsblock.c

```diff
@@ -35,7 +35,7 @@ const ActorInit Obj_Hsblock_InitVars = {
     (ActorFunc)ObjHsblock_Draw,
 };
 
-static f32 focusPos[] = { 85.0f, 85.0f, 0.0f };
+static f32 focusPoss[] = { 85.0f, 85.0f, 0.0f };
```

Comment:
```suggestion
static f32 sFocusPoss[] = { 85.0f, 85.0f, 0.0f };
```
Sorry, remember that static stuff starts with `s`.